docker-compose-old.yml

```diff
@@ -0,0 +1,252 @@
+version: '2.4'
```

Comment:
we probably don't need to keep this file around

Reply:
oh yeah, totally forgot to remove it